spec/requests/authentication_spec.rb

```diff
@@ -240,10 +240,17 @@
   end
 
   context "Token Based Authentication" do
-    %w(sql memory cache).each do |session_store|
-      context "when using a #{session_store} session store" do
-        before { stub_settings_merge(:server => {:session_store => session_store}) }
+    before do
+      # Keep this outside of the loop below with the `let` providing the value
+      # for the stub, otherwise it will not work as expected
+      stub_settings_merge(:server => {:session_store => session_store_value})      
+      TokenStore.token_caches.clear
```

Comment:
Can we put this in `EvmSpecHelper.clear_caches` instead?  That is called after every single spec.  Or is that not possible since that code is in core?

Reply:
Oh I was more thinking that, in general, any global caches should be cleared between specs...you never know if something else in the future will change to suddenly start using this token cache (or any other global cache).  I'm fine with this being tactical though.